### DISCUSSION TITLE
Help with loading paired images

### DISCUSSION THREAD

**Discussion by mecarill** [facebookresearch/detectron2#4109] [Q&A]
I'm looking to load paired images with detectron2. I am able to use register_coco_instances to register two datasets and build_detection_train_loader to allow them to be iterable but this would result in a shuffle for the images.

Ideally, I would want a way to load these paired images while shuffling the indexes, is there a way to do this?
